### PR TITLE
Update duckduckgo.po

### PR DESCRIPTION
**Pull request by itaizand**

### CODE REVIEW

**Comment by GioSensation** [comment]
Thanks for your contribution, @itaizand. We have a policy of not translating the brand name. Do you mind keeping `DuckDuckGo` in latin characters? Thanks!

**Comment by itaizand** [comment]
Yes ofcourse, sorry for the trouble. Is there a place where all the other
rules are written?

בתאריך יום ה׳, 11 בנוב׳ 2021, 12:50, מאת Emanuele Feliziani ‏<
***@***.***>:

**Comment by GioSensation** [comment]
Those are the only rules 🙂, and of course keeping the same amount of placeholders (if any) from the original string. But I agree we should add these rules to the README. I'll do it.

**Comment by itaizand** [comment]
Thanks a lot!
Do i need to manually fix the yimes i translated the brand's name?

בתאריך יום ה׳, 11 בנוב׳ 2021, 13:13, מאת Emanuele Feliziani ‏<
***@***.***>:

**Comment by GioSensation** [comment]
> Do i need to manually fix the yimes i translated the brand's name? 

Yes, please. I could do it using an automated translator, but I'd risk getting it wrong 🙂.

**Comment by itaizand** [comment]
Ofcourse! Thank for the clarification.
If it helps,
Duck duck go = דאק דאק גו

בתאריך יום ה׳, 11 בנוב׳ 2021, 15:35, מאת Emanuele Feliziani ‏<
***@***.***>:

File: locales/he_IL/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -5241,13 +5241,13 @@ msgctxt "Ads GDPR, internal use only. Do not change"
...
 "Microsoft's ad network"
-msgstr ""
+msgstr "פרטיות צפייה בפרסומות מוגנת ע"י duckduckgo. לחיצות על פרסומת מנוהלות ע"י רשת הפרסומות של מיקרוסופט"
 
 msgctxt "mobile promotion on desktop"
 msgid ""
 "Visit %sDuckDuckGo.com%s on your tablet or phone and follow the provided "
 "instructions."
-msgstr ""
+msgstr "בקר/י ב%sduckduckgo.com%s בטאבלט או בטלפון שלך ועקב/י אחר ההוראות שם. "
```
Comment (GioSensation) [inline locales/he_IL/LC_MESSAGES/duckduckgo.po:5250]:
```suggestion
msgstr "בקר/י ב%sDuckDuckGo.com%s בטאבלט או בטלפון שלך ועקב/י אחר ההוראות שם. "
```

File: locales/he_IL/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -5631,17 +5631,17 @@ msgstr "הסטנדרטי של YouTube"
...
 msgid "Your browser indicates if you've visited this link"
 msgstr "הדפדפן שלך נותן אינדיקציה אם ביקרת את קישור זה"
 
 msgctxt "new tab page"
-msgid ""
+msgid "כרטיסיה חדשה"
 "Your browser provides the list of most-visited sites. DuckDuckGo never has "
 "access to this data."
-msgstr ""
+msgstr "הדפדפן שלך מספק את רשימת האתרים הכי מקובלים. ל-duckduckgo לעולם אין גישה למידע הזה."
```
Comment (GioSensation) [inline locales/he_IL/LC_MESSAGES/duckduckgo.po:5644]:
```suggestion
msgstr "הדפדפן שלך מספק את רשימת האתרים הכי מקובלים. ל-DuckDuckGo לעולם אין גישה למידע הזה."
```

File: locales/he_IL/LC_MESSAGES/duckduckgo.po [hunk 3]
```diff
@@ -5654,12 +5654,12 @@ msgid ""
...
 
 msgctxt "cloudsave"
-msgid ""
+msgid "שמירה בענן"
 "Your passphrase generates a 512-bit key by using the Secure Hash Algorithm "
 "known as %sSHA-2%s. Only the key and associated settings file leave your "
 "browser. We save the settings file using the key as the name. DuckDuckGo "
 "never knows your passphrase."
-msgstr ""
+msgstr "הסיסמה שלך מייצרת מפתח בן 512-ביט ע"י שימוש באלגוריתם hash -%sSHA-2%s. רק המפתח וקובץ ההגדרות המזוהה איתו יוצאים מהדפדפן שלך. אנחנו שומרים את קובץ ההגדרות בעזרת מפתח באתו השר. duckduckgo לעולם לא יודעים את הסיסמה שלך.מאובטח הידוע כ"
```
Comment (GioSensation) [inline locales/he_IL/LC_MESSAGES/duckduckgo.po:5662]:
```suggestion
msgstr "הסיסמה שלך מייצרת מפתח בן 512-ביט עי שימוש באלגוריתם hash -%sSHA-2%s. רק המפתח וקובץ ההגדרות המזוהה איתו יוצאים מהדפדפן שלך. אנחנו שומרים את קובץ ההגדרות בעזרת מפתח באתו השר. DuckDuckGo לעולם לא יודעים את הסיסמה שלך.מאובטח הידוע כ"
```

File: locales/he_IL/LC_MESSAGES/duckduckgo.po [hunk 4]
```diff
@@ -5654,12 +5654,12 @@ msgid ""
 msgstr ""
 
 msgctxt "cloudsave"
-msgid ""
+msgid "שמירה בענן"
```
Comment (GioSensation) [inline locales/he_IL/LC_MESSAGES/duckduckgo.po:5657]:
```suggestion
msgid ""
```

File: locales/he_IL/LC_MESSAGES/duckduckgo.po [hunk 5]
```diff
@@ -5631,17 +5631,17 @@ msgstr "הסטנדרטי של YouTube"
...
-msgstr ""
+msgstr "מיקומך"
 
 #. Floating text over a tick next to search results which indicates if a link has been visited.
 msgid "Your browser indicates if you've visited this link"
 msgstr "הדפדפן שלך נותן אינדיקציה אם ביקרת את קישור זה"
 
 msgctxt "new tab page"
-msgid ""
+msgid "כרטיסיה חדשה"
```
Comment (GioSensation) [inline locales/he_IL/LC_MESSAGES/duckduckgo.po:5641]:
```suggestion
msgid ""
```

File: locales/he_IL/LC_MESSAGES/duckduckgo.po [hunk 6]
```diff
@@ -5089,12 +5089,12 @@ msgstr "מידות אימפריליות (פאונד, רגל, פרנהייט)"
 
 msgctxt "settingsvalue"
 msgid "US-based (pounds, feet, Fahrenheit)"
-msgstr ""
+msgstr "מבוסס על ארה"ב (פאונד, רגל, פרנהייט)"
```
Comment (GioSensation) [inline locales/he_IL/LC_MESSAGES/duckduckgo.po:5092]:
```suggestion
msgstr "מבוסס על ארהב (פאונד, רגל, פרנהייט)"
```

File: locales/he_IL/LC_MESSAGES/duckduckgo.po [hunk 7]
```diff
@@ -5117,7 +5117,7 @@ msgid "Under %sSTARTUP > Homepage%s enter: https://duckduckgo.com"
 msgstr "תחת %sהפעלה > עמוד הבית%s הכנס: https://duckduckgo.com"
 
 msgid "Under %sSearch Settings%s select %sDuckDuckGo%s"
-msgstr ""
+msgstr "תחת %sהגדרות חיפוש%s בחרו ב %sDuckDuckgo%s"
```
Comment (GioSensation) [inline locales/he_IL/LC_MESSAGES/duckduckgo.po:5120]:
```suggestion
msgstr "תחת %sהגדרות חיפוש%s בחרו ב %sDuckDuckGo%s"
```

File: locales/he_IL/LC_MESSAGES/duckduckgo.po [hunk 8]
```diff
@@ -1078,7 +1078,7 @@ msgstr "לחץ על התפריט הנפתח בתיבת החיפוש "
 msgid ""
 "Click the dropdown menu beside %sSearch engine used in the address bar%s and "
 "select %sDuckDuckGo%s."
-msgstr ""
+msgstr "תלחץ/י בתפריט לצד %sמנוע חיפוש בשימוש כתובת%s ותבחר/י %sדאקדאקגו%s"
```
Comment (GioSensation) [inline locales/he_IL/LC_MESSAGES/duckduckgo.po:1081]:
```suggestion
msgstr "תלחץ/י בתפריט לצד %sמנוע חיפוש בשימוש כתובת%s ותבחר/י %sDuckDuckGo%s"
```

**Comment by GioSensation** [comment]
@itaizand I managed to fix all syntax and format problems with this PR (see my previous comment for the future), but I'd ask you to please take a look at [my comment here](https://github.com/duckduckgo/duckduckgo-locales/pull/504/files#r838524799). It looks like there's some sort of mistake there. Please send me a message once you've fixed it so I can take another look and merge.

Thank you.

**Comment by GioSensation** [comment]
This PR has been stale for quite a while. I'm closing it now. Feel free to re-open or open a new one. Thank you.

**Pull request closed without merging**